source/jormungandr/jormungandr/authentication.py

```diff
@@ -105,6 +105,7 @@ def get_token():
         return auth
 
 
+@memory_cache.memoize(current_app.config['MEMORY_CACHE_CONFIGURATION'].get('TIMEOUT_AUTHENTICATION', 30))
 @cache.memoize(current_app.config['CACHE_CONFIGURATION'].get('TIMEOUT_AUTHENTICATION', 300))
```

Comment:
why we need to keep the old cache for this function? with the new docorator cache, the old cache is short-circuited, right?

Reply:
there is two layer of cache, in memory, and redis, the local cache has a very short timeout to prevent desynchronisation between jormungandr's instances, while the redis cache allow us to not depend too much of the database

Reply:
I mean... it seems that the result is always returned by memory_cache...

Reply:
we did talk about it directly, but for everyone else: these two layer have different cache duration, so an object can be in redis but not in memory